docker/init.sh

```diff
@@ -0,0 +1,48 @@
+#! /bin/bash
+
+set -e
+
+# data directory is required for caching the secret key in a file
+if [ "$DATA_DIR" == "" ]; then
+    echo -n "Error: DATA_DIR environment variable not set!"
+    echo "Are you sure you are running this script in a Docker container?"
+    exit 1
+fi
+
+SECRET_KEY_FILE="$DATA_DIR/secret_key"
+
+if [ ! -f "$SECRET_KEY_FILE" ]; then
+    echo ">>> Creating a new secret key file..."
+    install -m 0600 /dev/null "$SECRET_KEY_FILE"
+    SECRET_KEY=$(bundle exec rails secret)
```

Comment:
(Only in Rails 5 did they make `rake` and `rails` are interchangeable, which was always a source of confusion... OSEM is still on Rails 4)